apps/odyssey-stats/src/widget/modules.jsx

```diff
@@ -42,22 +46,30 @@ function ModuleCard( {
 						<div className="stats-widget-module__info">
 							{ error === 'not_active' && (
 								<button
+									className="jetpack-emerald-button"
 									disabled={ disabled }
-									onClick={ () => {
-										setDisabled( true );
-										activateProduct().catch( () => setDisabled( false ) );
-									} }
+									onClick={ onActivateProduct }
 								>
 									Activate
 								</button>
 							) }
-							{ /* TODO: add button to install plugins. */ }
-							{ error !== 'not_active' && info && (
-								<a href={ info.link } target="__blank">
-									{ info.text }
+							{ error === 'not_installed' && (
+								<button
+									className="jetpack-emerald-button is-secondary-jetpack-emerald"
+									disabled={ disabled }
+									onClick={ onActivateProduct }
+								>
+									Install
+								</button>
+							) }
+							{ error === 'invalid_key' && (
+								<a href={ manageUrl } target="_self">
+									Manage Akismet Key
```

Comment:
```suggestion
									Manage Akismet key
```

Nit: should be sentence case I think to match the heading above.